streams/contract_bytecode.proto

```diff
@@ -0,0 +1,39 @@
+syntax = "proto3";
+
+package proto;
+
+/*-
+ * ‌
+ * Hedera Network Services Protobuf
+ * ​
+ * Copyright (C) 2018 - 2022 Hedera Hashgraph, LLC
+ * ​
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *      http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ * ‍
+ */
+
+option java_package = "com.hederahashgraph.api.proto.java";
+option java_multiple_files = true;
+
+message ContractBytecode {
+
+  /**
+  * Contract bytecode during deployment
+  */
+  bytes initcode = 1;
+
+  /**
+  * Contract bytecode after deployment
+  */
+  bytes runtime_bytecode = 2;
```

Comment:
I think we would also need the `ContractID contract_id` here as well for migrations. For non-migration we can use the associated consensus timestamp to link it to the contract create. 

For migrations, the consensus timestamp is for a random transaction after startup and not associated with the contract create. So we would need the contract ID here or we would need the consensus timestamp to be linked to some contract create synthetic transaction.